lite2/client.go

```diff
@@ -705,11 +705,11 @@ func (c *Client) sequence(
 	now time.Time) error {
 	// 1) Verify any intermediate headers.
 	var (
-		interimHeader   *types.SignedHeader
 		interimNextVals *types.ValidatorSet
+		interimHeader   *types.SignedHeader
```

Comment:
why move down?

Reply:
what do you mean?

Reply:
![Screenshot 2020-02-21 at 12 14 14](https://user-images.githubusercontent.com/1282182/75030112-b9184780-54a3-11ea-9b9b-551631c498dc.png)

Reply:
Ah I see. I removed it at first and then I added it back later. Does it make a difference

Reply:
consistency makes a great difference when other developers reading your code